lib/matplotlib/backends/backend_pgf.py

```diff
@@ -429,7 +432,8 @@ def __init__(self, figure, fh, dummy=False):
                               UserWarning)
                 self.__dict__["draw_image"] = lambda *args, **kwargs: None
 
-    def draw_markers(self, gc, marker_path, marker_trans, path, trans, rgbFace=None):
+    def draw_markers(
+            self, gc, marker_path, marker_trans, path, trans, rgbFace=None):
```

Comment:
Kinda prefer to wrap at the kwarg (dunno why this didn't comment before.)

Reply:
your wish is my command :p